main.cpp

```diff
@@ -51,85 +51,96 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
+//  1) 
+//  Noun: tape recorder
+//  action 1: play tape half speed
+tapeRecorder.playFullSpeed();
+//  action 2: change tape direction
+tapeRecorder.changeTapeDirection();
+//  action 3: record input
+tapeRecorder.record();
 
-//  action 2:
-
-//  action 3:
  
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: garbage truck
+//  action 1: crushing trash
+garbageTruck.crushTrash();
+//  action 2: dumping trash 
+garbageTruck.dumpGarbage();
+//  action 3: stoping  
+garbageTruck.stop();
  
 //  3)
-//  Noun:
-//  action 1:
+//  Noun: guitarist 
+//  action 1: play a chord
+guitarist.playChord();
+//  action 2: make noise with pedals 
+guitarist.makeNoise();
+//  action 3: sing harmonies 
+guitarist.singHarmonies();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.